nc/source/component/Collider.cpp

```diff
@@ -4,38 +4,75 @@
 
 #ifdef NC_EDITOR_ENABLED
 #include "ui/editor/Widgets.h"
-#endif
 
 namespace
 {
-    const auto ColliderMesh = std::string{"project/Models/cube.fbx"};
-    auto GetColliderMaterial = nc::graphics::Material::CreateMaterial<nc::graphics::TechniqueType::Wireframe>;
+    const auto CubeMeshPath = std::string{"project/Models/cube.fbx"};
+    auto CreateMaterial = nc::graphics::Material::CreateMaterial<nc::graphics::TechniqueType::Wireframe>;
+
+    auto CreateBoxModel()
```

Comment:
See comments about not using "Model" as a name. What about "BoundingBox" or "BoundingSphere" "BoundingCapsule" etc